app/javascript/components/payment-error-button.vue

```diff
@@ -159,6 +162,18 @@ export default {
     getUsers() {
       return invoiceApi.getSlackUsers();
     },
+    notifySellersOfDebt(products) {
+      const message = `${this.message.displayNameNormalized} acaba de fiar un producto tuyo!`;
+      products.forEach(prod => {
+        invoiceApi.getSeller(prod).then((seller) => {
+          const messageToDebtor = `${this.message.displayNameNormalized}, acabas de fiar un producto de ${seller.user.name}`
```

Comment:
Line 169 exceeds the maximum line length of 120  max-len<br>Missing semicolon                                semi